contrib/ProtobufLogger.py

```diff
@@ -144,16 +144,20 @@ def printSummary(self, msg, typestr):
         if requestor:
             requestorstr = ' (' + requestor + ')'
 
-        print('[%s] %s of size %d: %s%s -> %s (%s), id: %d, uuid: %s%s' % (datestr,
-                                                                           typestr,
-                                                                           msg.inBytes,
-                                                                           ipfromstr,
-                                                                           requestorstr,
-                                                                           iptostr,
-                                                                           protostr,
-                                                                           msg.id,
-                                                                           messageidstr,
-                                                                           initialrequestidstr))
+        deviceId = binascii.hexlify(bytearray(msg.deviceId))
+        requestorId = msg.requestorId
+
+        print('[%s] %s of size %d: %s%s -> %s (%s), id: %d, uuid: %s%s '
+                  'requestorid: %s deviceid: %s' % (datestr,
+                                                    typestr,
+                                                    msg.inBytes,
+                                                    ipfromstr,
+                                                    requestorstr,
+                                                    iptostr,
+                                                    protostr,
+                                                    msg.id,
+                                                    messageidstr,
+                                                    initialrequestidstr))
```

Comment:
requestorid and deviceid are missing in this parameter list

Reply:
`[2017-08-28 16:46:49.461187] Response of size 50: 127.0.0.1 -> 127.0.0.1 (UDP), id: 47036, uuid: f2b4bcba0b144a09b9a49b234770f142 requestorid: pietje deviceid: 414243444546`

With:
```diff
-                                                    initialrequestidstr))
+                                                    initialrequestidstr, requestorId, deviceId))
```